src/dns/aardvark.rs

```diff
@@ -220,7 +221,11 @@ impl Aardvark {
         }
 
         for entry in &entries {
-            let path = Path::new(&self.config).join(entry.network_name);
+            let mut path = Path::new(&self.config).join(entry.network_name);
+            if entry.is_internal {
+                let _ = std::fs::remove_file(&path);
+                path = Path::new(&self.config).join(entry.network_name.to_owned() + "%int");
+            }
```

Comment:
This is breaking backwards compat.  If there are already running containers then you just delete all info from them.
The proper approach must be to rename the old path when it exists IMO. That way we migrate it automatically.